metrics/cpu_syscall.go

```diff
@@ -31,5 +31,5 @@ func getProcessCPUTime() int64 {
 		log.Warn("Failed to retrieve CPU time", "err", err)
 		return 0
 	}
-	return int64(usage.Utime.Sec+usage.Stime.Sec)*100 + int64(usage.Utime.Usec+usage.Stime.Usec)/10000 //nolint:unconvert
+	return usage.Utime.Sec + usage.Stime.Sec*100 + int64(usage.Utime.Usec+usage.Stime.Usec)/10000 //nolint:unconvert
```

Comment:
You messed up the math, parenthesis was needed

Reply:
Ouch, Fixed! 🙏